token/cli/src/output.rs

```diff
@@ -257,6 +259,20 @@ impl fmt::Display for CliTokenAccount {
             writeln!(f, "* Please run `spl-token gc` to clean up Aux accounts")?;
         }
 
+        if self.has_permanent_delegate {
+            writeln!(f)?;
+            writeln!(
+                f,
+                "* {} ",
+                style("This token mint has a permanent delegate!").bold()
+            )?;
+            writeln!(f, "  This means the mint owner or their agent may withdraw {} funds from this account at {} time.",
```

Comment:
I got a little hung up on what "mint owner or their agent" means. Would this be too obscure/complex?

```suggestion
            writeln!(f, "  This means the mint's permanent delegate authority may withdraw {} funds from this account at {} time.",
```

Reply:
perhaps "This means the person who controls the mint may..."?

Reply:
Well, it might not be a single person, right? And it might not be the same authority that controls minting or freezing or closing.
But I'm not sure if greater precision and accuracy is worth it.

Reply:
yea my intended audience is nontechnicals and im trying to communicate "if you put a lot of money in this and dont know what this thing means then someone is trying to steal from you." i imagine permanent delegates are going to be used for rugs unless there is a lot of signposting about what its for (this is going to be on wallets, mostly, unfortunately)

maybe "This means the mint may withdraw"?

Reply:
> maybe "This means the mint may withdraw"?

I like this best of the options so far. :shipit: ?